deb/openmediavault/debian/openmediavault.postinst

```diff
@@ -289,6 +298,9 @@ case "$1" in
 			LANG=C.UTF-8 omv-salt deploy run --no-color --quiet apt cronapt || :
 		fi
 		if dpkg --compare-versions "$2" lt-nl "5.6.4"; then
+		  useradd --system --user-group --no-create-home \
+		    --shell '/usr/sbin/nologin' openmediavault-notify || :
+			omv_module_set_dirty postfix
```

Comment:
The indentation is inconsistent here.

Reply:
Another strange IDE behavior. The IDE is mixes tabs and blanks wildly.